BedrockPlugin.cpp

```diff
@@ -46,6 +46,7 @@ BedrockPlugin* BedrockPlugin::getPlugin(const string& name) {
 void BedrockPlugin::enable(bool enabled) { _enabled = enabled; }
 bool BedrockPlugin::enabled() { return _enabled; }
 string BedrockPlugin::getName() { SERROR("No name defined by this plugin, aborting."); }
+const STable& BedrockPlugin::getInfo() { return pluginInfo; }
```

Comment:
Nab just something to keep in mind, but doesn't this mean that Plugins will have to be married to STable (and thus libstuff) in order to report Status?